blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,244 @@
+    "johndoe123" : 0,
+    "larry_l12" : 1,
+    "splashbro30" : 2,
+}
+
+def _pull_data(request):
+    with open('banking_blueprint/data/sample_user_data.json') as f:
+        global user_data 
+        data = json.load(f)
+        if request.context.get('user_name'):
+            user_name = request.context.get('user_name')
+            user = sample_users[user_name]
+        else:
+            user = random.choice([0, 1, 2])
+
+        user_data = data[user]
+
+def _get(key):
+    return user_data[key]
+
+def _put(key, value):
+    global user_data
+    user_data[key] = value
+
+def check_value(responder):
+    return _get(responder.slots['origin']) >= responder.slots['amount']
+
+#Slot filling forms 
+
+entity_form = {
+    'entities':[
+        FormEntity(
+            entity='account_type',
+            role='origin',
+            responses=['Sure. Transfer from which account?']),
+        FormEntity(
+            entity='account_type',
+            role='dest',
+            responses=['To which account?']),
+        FormEntity(
+            entity='sys_amount-of-money',
+            responses=['And, how much do you want to transfer?'])
+
+        ],
+        'exit_keys' : ['cancel', 'restart', 'exit', 'reset', 'no', 'nevermind', 'stop', 'back', 'help', 'stpo it', 'go back'
+            'new task', 'other', 'return'],
+        'exit_msg' : 'A few other banking tasks you can try are, reporting a fraudelent charge and setting up AutoPay',
+        'max_retries' : 1
+}
+
+balance_form = {
+    'entities':[
+    FormEntity(
+        entity='account_type',
+        responses=['Sure. for which account?'])
+    ],
+    'exit_keys' : ['cancel', 'restart', 'exit', 'reset', 'no', 'nevermind', 'stop', 'back', 'help', 'stop it', 'go back'
+            'new task', 'other', 'return'],
+    'exit_msg' : 'A few other banking tasks you can try are, ordering checks and paying bills',
+    'max_retries' : 1
+}
+
+#Dialogue state handlers 
+
+@app.handle(intent='greet')
+def greet(request, responder):
+    if not user_data: 
+        _pull_data(request)
+    responder.slots['name'] = _get('first_name')
+    replies = ['Hi {name}, How can I help with your banking tasks? You can try transferring money between accounts or paying a bill.',
+     'Hello {name}, thanks for choosing MindMeld Bank. You can do things like, ask for your routing number, order checks, and check bill due dates.',
+     'Welome to MindMeld Bank {name}, How can I help with your banking tasks? You can try reporting a fraud charge or a lost credit card.',
+     'Thanks for using MindMeld Bank {name}! What would you like to do today? A few things I can help with are, checking balances, paying off your credit card, and setting up a new card.']
+    responder.reply(replies)
+
+@app.handle(intent='exit')
+def say_goodbye(request, responder):
+    responder.reply(['Bye', 'Goodbye', 'Have a nice day.'])
+
+
+@app.handle(intent='lost_creditcard')
+def faq_lost_creditcard_handler(request, responder):
```

Comment:
For all of the FAQ responses, it would be nice to update the language to acknowledge that we've noted the user's intent before telling them to visit the website/branch/email. 

e.g. instead of
`'If your card is lost or stolen, or you think someone used your account without permission, tell us immediately by calling 1-800-432-3117.'`
we could say
`I've noted that your card may have been lost or stolen. Please follow up immediately by calling 1-800-432-3117 or clicking the link in the email sent to user@gmail.com`
We can also add emails to the user data and populate the email values from there.